### PR TITLE
Stac 10 merge 41 to master

### PR DESCRIPTION
**Pull request by gggina**
Merges assets changed by gitbook to master ready for automated pull requests.

### CODE REVIEW

File: develop/scripting/script-apis/prediction.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Predict metrics for any metric query coming from any data source.
 **Args:**
 
 * `predictorName` - name of prediction preset. Current available predictors: `linear`, `hmn` and `fft`.
-* `horizon` - how much future to predict. The horizon is specified in the [duration format](time.mc).
+* `horizon` - how much future to predict. The horizon is specified in the [duration format](https://github.com/StackVista/stackstate-docs/tree/e642f99b5f9f4a7b201b558ae44375317f5974c5/develop/scripting/script-apis/time.mc).
```
Comment (mpvvliet) [inline develop/scripting/script-apis/prediction.md:16]: Is this URL correct?

File: configure/subject_configuration.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ kind: Documentation
 
 ## Link your existing LDAP to StackState RBAC
 
-StackState authentication is configured in a [config file](authentication.md) that contains already predefined roles for the Guest user \(very limited permission level\) and the Admin user \(full permission level\). To change the configuration to use LDAP authentication, check out this document: [How to configure LDAP authentication](how_to_configure_ldap_authentication.md).
+StackState authentication is configured in a [config file](https://github.com/StackVista/stackstate-docs/tree/e642f99b5f9f4a7b201b558ae44375317f5974c5/configure/authentication.md) that contains already predefined roles for the Guest user \(very limited permission level\) and the Admin user \(full permission level\). To change the configuration to use LDAP authentication, check out this document: [How to configure LDAP authentication](how_to_configure_ldap_authentication.md).
```
Comment (mpvvliet) [inline configure/subject_configuration.md:10]: Is this the way to refer to files?

File: getting_started.md [hunk 3]
```diff
@@ -11,17 +11,17 @@ Hi! So, you've just installed StackState and you are ready to get started.
...
+![StackPacks overview](.gitbook/assets/01_stackpacks%20%281%29.png)
 
 ## Explore topology
 
 After installing one or more StackPacks, you can go to the Explore Mode to explore your IT landscape or visit a specific [view](use/views.md) from your installed StackPacks.
 
 StackState visualizes components by the layer and domain they are placed in. These are logical groupings of components. Layers are displayed on the vertical axis. Domains are displayed on the horizontal axis.
 
-You can change which part of the landscape you are viewing \(e.g. layers and domains\) with the [filter options](use/filtering.md) on the left side bar, or by [hovering over a component](use/topology-Perspective.md#interactive-navigation).
+You can change which part of the landscape you are viewing \(e.g. layers and domains\) with the [filter options](https://github.com/StackVista/stackstate-docs/tree/e642f99b5f9f4a7b201b558ae44375317f5974c5/use/filtering.md) on the left side bar, or by [hovering over a component](https://github.com/StackVista/stackstate-docs/tree/e642f99b5f9f4a7b201b558ae44375317f5974c5/use/topology-Perspective.md#interactive-navigation).
```
Comment (mpvvliet) [inline getting_started.md:22]: These should probably be relative links

File: setup/authentication.md [hunk 4]
```diff
@@ -21,7 +21,7 @@ StackState ships with the default user roles **guest** and **administrator**:
 * **guest users** - able to see information but make no changes
 * **administrators** - able to see and change all configuration
 
-It is also possible to add more roles, see the [Roles (RBAC)](../configure/how_to_set_up_roles.md) and the other RBAC documentation pages under [configure](../configure.md)
+It is also possible to add more roles, see the [Roles \(RBAC\)](../configure/how_to_set_up_roles.md) and the other RBAC documentation pages under [configure](https://github.com/StackVista/stackstate-docs/tree/e642f99b5f9f4a7b201b558ae44375317f5974c5/configure.md)
```
Comment (mpvvliet) [inline setup/authentication.md:24]: Relative link?

File: setup/backup_restore/file.md [hunk 5]
```diff
@@ -8,15 +11,16 @@ To backup and restore StackState configuration and topology information we can u
 
 ### Backup
 
-In order to backup StackGraph's data on a [production setup](../linux_install/production-installation.md) you need to: 1. Have StackGraph up and running 2. Stop StackState node [Starting and Stopping StackState](../linux_install/production-installation#starting-and-stopping-stackstate) `sudo systemctl stop stackstate.service` 3. Use one of the following commands to start the backup process:
+In order to backup StackGraph's data on a [production setup](../linux_install/production-installation.md) you need to: 1. Have StackGraph up and running 2. Stop StackState node [Starting and Stopping StackState](https://github.com/StackVista/stackstate-docs/tree/e642f99b5f9f4a7b201b558ae44375317f5974c5/setup/linux_install/production-installation/README.md#starting-and-stopping-stackstate) `sudo systemctl stop stackstate.service` 3. Use one of the following commands to start the backup process:
```
Comment (mpvvliet) [inline setup/backup_restore/file.md:14]: Relative link?

**Review by mpvvliet — COMMENTED** [review]
Overall, there are a bunch of github URL links directly to commits, those don't seem right to me

File: configure/subject_configuration.md (hunk 2 above)
Comment (gggina) [reply configure/subject_configuration.md:10]: no this is wrong. It does this when the link is wrong. I will check and update them. And try to figure out why it happened

**Review by mpvvliet — APPROVED** [review]

**Pull request merged**